gtdynamics/cablerobot/src/cdpr_planar.py

```diff
@@ -151,18 +151,14 @@ def dynamics_factors(self, ks=[]):
         """
         dfg = gtsam.NonlinearFactorGraph()
         for k in ks:
-            wf = gtd.WrenchFactor(
-                    gtd.internal.TwistKey(self.ee_id(), k).key(),
-                    gtd.internal.TwistAccelKey(self.ee_id(), k).key(),
+            # TODO(yetong): Use EqualityConstraint.createFactor when wrapped.
+            dfg.add(gtd.WrenchFactor(self.costmodel_wrench, self.eelink(),
                     [
                         gtd.internal.WrenchKey(self.ee_id(), 0, k),
                         gtd.internal.WrenchKey(self.ee_id(), 1, k),
                         gtd.internal.WrenchKey(self.ee_id(), 2, k),
                         gtd.internal.WrenchKey(self.ee_id(), 3, k)
-                    ],
-                    gtd.internal.PoseKey(self.ee_id(), k).key(),
-                    self.costmodel_wrench, self.eelink().inertiaMatrix(), self.params.gravity)
-            dfg.push_back(wf)
+                    ], k, self.params.gravity))
```

Comment:
Thanks for the ping.  Just for context, the primary reason I use these factors is because my CableFactors (`CableLengthFactor`, `CableVelocityFactor`, `CableTensionFactor`) are not quite the same as `PoseFactor`/`TwistFactor`/`TwistAccelFactor`/`TorqueFactor` so therefore I cannot create a "Cable Joint" and therefore cannot use the `Robot` machinery :( figuring out a nice way to incorporate these is on my TODO list.

After your guys' long discussion, it seems the conclusion is that WrenchFactor is going to no longer be part of the python interface?  If so, could you maybe create a `TempWrenchFactor` similar to `TempPoseFactor` to tide me over until equality stuff is ready for me to migrate everything over?

Also, I _think_ it could also be something simple and wouldn't need to retain the entire old factor as in `TempPoseFactor`.  Something like:
```c++
gtsam::NonlinearFactor::shared_ptr TempWrenchFactor(args) {
    return boost::make_shared<WrenchFactor>(args);
}
```
gtdynamics.i:
```c++
gtsam::NonlinearFactor::shared_ptr TempWrenchFactor(args);
```
or something (not tested)

Reply:
I suggest meeting briefly meeting with Yetong

Reply:
A temporary solution is to keep the WrenchFactor wrapper to make the test working, and making an Issue and a TODO to resolve that in the future.